acmwebsite/controllers/root.py

```diff
@@ -47,10 +48,11 @@ class RootController(BaseController):
     mailinglist = MailingListController()
     u = UsersController()
     m = MeetingsController()
-    s = SurveysController()
+    survey = SurveysController()
     schedule = ScheduleController()
     error = ErrorController()
     contact = ContactController()
+    projects = ProjectsController()
```

Comment:
This controller needs to be renamed. It is getting too confusing. So far, a `ThingsController` has been responsible for looking up a thing, not for listing all things. This controller should really be called `ProjectListController`.